bin/sim_interpreter.py

```diff
@@ -485,9 +487,17 @@ def token_utilizer(self, instruction_token, name, start_address, size_of_target,
         elif instruction_token == 44: #SUBF
             pass
         elif instruction_token == 45: #SUBR
-            pass
+           r2 = self.registers.get_register(arguments[1])
+           r1 = self.registers.get_register(arguments[0])
+           self.registers.set_register(self.registers.get_register(arguments[1]), sub_hex(r2, r1))
```

Comment:
Does this work with negative numbers? Also set_register takes a letter and the value you are trying to set, the arguments you provided wouldn't work